CMakeLists.txt

```diff
@@ -249,4 +261,7 @@ set_tests_properties(cargo_build_rerun PROPERTIES RUN_SERIAL TRUE)
 
 add_subdirectory(book)
 add_subdirectory(examples)
-add_subdirectory(tests)
+
+if(NOT BUILD_WASM)
```

Comment:
i wonder if a `BUILD_TESTS` is better, as then people can skip building tests when building non-wasm builds?